doc/classes/RenderingServer.xml

```diff
@@ -4115,6 +4129,14 @@
 		</constant>
 		<constant name="VIEWPORT_DEBUG_DRAW_MOTION_VECTORS" value="25" enum="ViewportDebugDraw">
 		</constant>
+		<constant name="VIEWPORT_VRS_DISABLED" value="0" enum="ViewportVRSMode">
+		</constant>
+		<constant name="VIEWPORT_VRS_TEXTURE" value="1" enum="ViewportVRSMode">
+		</constant>
+		<constant name="VIEWPORT_VRS_XR" value="2" enum="ViewportVRSMode">
+		</constant>
+		<constant name="VIEWPORT_VRS_MAX" value="3" enum="ViewportVRSMode">
+		</constant>
```

Comment:
Just to save you trouble coming up with a description for this one, here's our canonical template for `_MAX` constants:
```suggestion
		<constant name="VIEWPORT_VRS_MAX" value="3" enum="ViewportVRSMode">
			Represents the size of the [enum ViewportVRSMode] enum.
		</constant>
```